x/posts/client/rest/tx.go

```diff
@@ -44,7 +44,16 @@ func createPostHandler(cliCtx context.CLIContext) http.HandlerFunc {
 			return
 		}
 
-		msg := types.NewMsgCreatePost(req.Message, parentID, req.AllowsComments, req.Subspace, req.OptionalData, addr, req.CreationTime)
+		var msg types.MsgCreatePost
+		msg = types.NewMsgCreateTextPost(req.Message, parentID, req.AllowsComments, req.Subspace, req.OptionalData,
```

Comment:
If this `msg` variable initialization exceeds the line limit, can we put each value on its line? I think it would make it more easy to read.